### ISSUE TITLE
[Feature Request]: Can I check the loss when finetuning  Embedding model?

### ISSUE THREAD

**Issue by Lauorie** [run-llama/llama_index#9171]
### Feature Description

I was using [finetune_embedding.ipynb](https://github.com/run-llama/llama_index/blob/main/docs/examples/finetuning/embeddings/finetune_embedding.ipynb) finetuning an Embedding model, but found no where to return loss curve, it should include train loss and val loss. So how can I get them? Thanks.

**Comment by logan-markewich** [comment]
@Lauorie looking at the sentence transformer docs, I don't really see a way? https://www.sbert.net/docs/training/overview.html#sentence_transformers.SentenceTransformer.fit

**Comment by Lauorie** [comment]
> @Lauorie looking at the sentence transformer docs, I don't really see a way? https://www.sbert.net/docs/training/overview.html#sentence_transformers.SentenceTransformer.fit
> 
> Under the hood, we just do this
> 
> ```
>     def finetune(self, **train_kwargs: Any) -> None:
>         """Finetune model."""
>         self.model.fit(
>             train_objectives=[(self.loader, self.loss)],
>             epochs=self.epochs,
>             warmup_steps=self.warmup_steps,
>             output_path=self.model_output_path,
>             show_progress_bar=self.show_progress_bar,
>             evaluator=self.evaluator,
>             evaluation_steps=self.evaluation_steps,
>         )
> ```

OKay, you still own me a [request](https://github.com/run-llama/llama_index/issues/8884#issuecomment-1823819195) my friend ^ _ ^
